seed-job/parser.py

```diff
@@ -25,7 +25,7 @@
         "-p APP_ID={} ".format(entry["app-id"]) + \
         "-p JOB_ID={} ".format(entry["job-id"]) + \
         "-p REGISTRY_URL={}".format(
-            os.environ.get("REGISTRY_URL") or "registry.centos.org"
+            os.environ.get("REGISTRY_URL")
```

Comment:
How are we going to set an environment variable in a Jenkins slave pod that is spun up dynamically?